src/java/org/apache/cassandra/tools/LoaderOptions.java

```diff
@@ -173,8 +173,10 @@ static class Builder
         int storagePort;
         int sslStoragePort;
         EncryptionOptions clientEncOptions = new EncryptionOptions();
+        EncryptionOptions.Builder clientEncOptionsBuilder = new EncryptionOptions.Builder(clientEncOptions);
         int connectionsPerHost = 1;
         EncryptionOptions.ServerEncryptionOptions serverEncOptions = new EncryptionOptions.ServerEncryptionOptions();
+        EncryptionOptions.ServerEncryptionOptions.Builder serverEncOptionsBuilder = new EncryptionOptions.ServerEncryptionOptions.Builder(serverEncOptions);
```

Comment:
@maulin-vasavada this is not used anywhere

Reply:
Thanks @smiklosovic for pointing it out. It led to a little bigger discovery :) I realized I had to make little more changes to the `LoaderOptions#Builder`. I made those changes. However, I am thinking if there is a way to test this class. Currently there are no tests I can run to validate (otherwise most likely I could have caught this missing changes). Any suggestions for tests?

Reply:
I added more test to validate the encryption options override that can catch what my pervious changes missed.